src/org/zaproxy/zap/extension/jxbrowser/ZapBrowserPanel.java

```diff
@@ -87,6 +89,13 @@ public Browser getBrowser() {
                 browser = new Browser();
             }
         }
+        if (browser.getConsoleListeners().isEmpty()) {
+            browser.addConsoleListener(new ConsoleListener() {
+                public void onMessage(ConsoleEvent event) {
+                    LOGGER.info("jxBrowser Console Event - Level: " + event.getLevel() + " - Message: " + event.getMessage());
```

Comment:
We only use 'info' for very significant events. I think 'debug' would be more appropriate (for the ZAP log).
Other than that it looks good :)

Reply:
Switched to debug logging. Not sure how to enable it for testing.

Reply:
Just manually edit your ./ZAP_D/log4j.properties to change these 2 lines to DEBUG:
https://github.com/zaproxy/zaproxy/blob/develop/src/xml/log4j.properties#L16-L17
and restart ZAP :)

Reply:
👍 for some reason I thought there was a commandline option.

Cool, well this is logging stuff now when debug is enabled.

Reply:
You can also change the log level dynamically, for example, with a JavaScript Stand Alone script, e.g.
```JavaScript
org.apache.log4j.Logger.getLogger("org.zaproxy.zap.extension.jxbrowser.ZapBrowserPanel").setLevel(org.apache.log4j.Level.DEBUG)
```